lib/bootloader_setup.pm

```diff
@@ -151,7 +151,7 @@ sub add_custom_grub_entries {
         my $script_new_esc = $script_new =~ s~/~\\/~rg;
         assert_script_run("cp -v $script_old $script_new");
 
-        my $cmd = "sed -i -e 's/\\(args=.\\)\\(\\\$4\\)/\\1$grub_param \\2/'";
+        my $cmd = "sed -i -e 's/\\(args=.\\)\\(\\\$4\\)/\\1\\2 $grub_param/'";
```

Comment:
I prefer following change then you have same order of GRUB_PARAM, otherwise you will get reverse orde.
my $cmd = "sed -i -e 's/\\(args=.\\)\\(\\\$4.*\[^\"\]\\)/\\1\\2 $grub_param/'";

Reply:
Could you please show the example what it would fix and which reverse order do you mean?

We are changing the line  `args="$4"`  to ` args="$4 debug_pagealloc=on"` and without this PR line would be  `args="debug_pagealloc=on $4"`

original: `args="$4"` 
old behavior:  `args="debug_pagealloc=on $4"`
new behavior: `args="$4 debug_pagealloc=on"`

Reply:
Imagine the original string looks like this: `args="foo $4 bar"`. If `\2` doesn't include everything up to the closing quotation mark, you'll get `args="foo $4 debug_pagealloc=on bar"`

Reply:
Latest VR: https://openqa.suse.de/tests/11897495#step/install_ltp/184